src/Weather.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class Weather extends Component {
+  state = {
+    temp: "",
+    pic: "",
+    description: "",
+  };
+
+  componentDidMount() {
+    fetch(
+      `http://api.weatherstack.com/current?access_key=${process.env.REACT_APP_WEATHER}&query=Toronto&units=m`
+    )
+      .then((res) => {
+        return res.json();
+      })
+      .then((data) =>
+        this.setState({
+          pic: data.current.weather_icons[0],
+          description: data.current.weather_descriptions[0],
+          temp: data.current.temperature,
+        })
+      )
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+
+  render() {
+    return (
+      <div className="ui card" id="weather">
+        <div class="content">
```

Comment:
Is there a reason you've chosen to 'nest' things so deeply? It might be my inexperience with react, but it seems like the content div and image div aren't adding much, and they aren't referred to in the css file, which is where I would expect the utility to come from in this kind of presentational component.

Reply:
using semantic ui to add css.